src/index.js

```diff
@@ -0,0 +1,262 @@
+    * @param {object} { page: 1, size: 2 } the query param in the format of key pair.
+    * @return {Method} Provide method list in array format.
+    */
+    function byName(queryParam) {
+      return GetAiEngineByName(new Headers(module.exports.api_key, module.exports.organization), queryParam, name);
+    }
+
+    /**
+    * Summary: take provider or engine name to perfom GET AI Engines.
+    * 
+    * This mehtod required Header `Authorization: Bearer {bearer_id}`, you can obtain `bearer_id` using `.Auth().accessTokenLogin()` method.
+    * AI Provider name (case sensitive), Only one parameter is required either `Provider Name` or `Engine`
+    *
+    * @param {string} Provider some provider like `openai`, `cohere`, `mantium`, `OpenAI`, `Cohere`, `Mantium`
+    * @param {string} Engine AI Engine name.
+    * @return {method} This return the method `.listMethods(queryParam)`.
+    */
+    function main(e) {
+      name = e;
+      return {
+        all: all,
+        byProvider: byProvider,
+        byName: byName
+      };
+    }
+    main.all = all;
+    main.byProvider = byProvider;
+    main.byName = byName;
+
+    return main;
+  })(),
+
+  Tags: (function () {
+
+    /**
+    * Summary: Get all of the tags for your selected organization.
+    * 
+    * This mehtod required Header `Authorization: Bearer {bearer_id}`, you can obtain `bearer_id` using `.Auth().accessTokenLogin()` method.
+    * @param {object} { 'page': 1, 'size': 20 };
+    * 
+    * @return {Method} Provide method list in array format.
+    */
+    function list(data) {
+      return Tag(new Headers(module.exports.api_key, module.exports.organization), { type: 'list', method: 'GET', queryParam: data });
+    }
+
+    /**
+    * Summary: Create a Tag.
+    * 
+    * This mehtod required Header `Authorization: Bearer {bearer_id}`, you can obtain `bearer_id` using `.Auth().accessTokenLogin()` method.
+    * @param {object} { name: 'tag name', description: 'Some description' };
+    * 
+    * @return {Method} Provide method list in array format.
+    */
+    function create(data) {
+      return Tag(new Headers(module.exports.api_key, module.exports.organization), Object.assign({ type: 'item', method: 'POST' }, data));
+    }
+
+    /**
+    * Summary: Update a Tag.
+    * 
+    * This mehtod required Header `Authorization: Bearer {bearer_id}`, you can obtain `bearer_id` using `.Auth().accessTokenLogin()` method.
+    * @param {object} { id: 'tagid', name: 'tag name', description: 'Some description' };
+    * 
+    * @return {Method} Provide method list in array format.
+    */
+    function update(data) {
+      return Tag(new Headers(module.exports.api_key, module.exports.organization), Object.assign({ type: 'item', method: 'PATCH' }, data));
+    }
+
+    /**
+    * Summary: Get details about a specific tag.
+    * 
+    * This mehtod required Header `Authorization: Bearer {bearer_id}`, you can obtain `bearer_id` using `.Auth().accessTokenLogin()` method.
```

Comment:
* Spelling Mistake: mehtod is method
* This method requires

Reply:
Done